services/src/main/java/org/keycloak/broker/oidc/mappers/AbstractClaimToGroupMapper.java

```diff
@@ -67,22 +73,16 @@ public void updateBrokeredUser(KeycloakSession session, RealmModel realm, UserMo
     protected abstract boolean applies(final IdentityProviderMapperModel mapperModel,
             final BrokeredIdentityContext context);
 
-    /**
-     * Obtains the {@link GroupModel} corresponding the group configured in the specified
-     * {@link IdentityProviderMapperModel}.
-     * If the group doesn't exist, this method throws an {@link IdentityBrokerException}.
-     *
-     * @param realm a reference to the realm.
-     * @param mapperModel a reference to the {@link IdentityProviderMapperModel} containing the configured group.
-     * @return the {@link GroupModel}
-     * @throws IdentityBrokerException if the group doesn't exist.
-     */
     private GroupModel getGroup(final RealmModel realm, final IdentityProviderMapperModel mapperModel) {
-        GroupModel group = KeycloakModelUtils.findGroupByPath(realm, mapperModel.getConfig().get(ConfigConstants.GROUP));
+        String groupPath = mapperModel.getConfig().get(ConfigConstants.GROUP);
+        GroupModel group = KeycloakModelUtils.findGroupByPath(realm, groupPath);
 
         if (group == null) {
-            throw new IdentityBrokerException("Unable to find group: " + group.getId());
+            LOG.warnf("Unable to find group by path '%s' for mapper '%s' on realm '%s'.", mapperModel.getName(),
```

Comment:
IMO it worth to create LOG inside the class `AbstractClaimToGroupMapper` rather then `AbstractClaimMapper` ? The reason is, that admin would see the warning from class `AbstractClaimMapper`, but he won't know which mapper it really is.

Reply:
@mposolda Thanks for the hint, adjusted as you proposed.